contracts/utils/EventCreation.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract SimpleTransaction {
```

Comment:
nit: like i don't have strong opinions on this or anything but we did kinda have a planet naming thing going on for example contracts

but I can't emphasize enough. I don't have any strong opinions on this. 

But we did have a planet thing going on

Reply:
Thank you. Will rename to mercury